packages/Webkul/Shop/src/Resources/views/checkout/cart/index.blade.php

```diff
@@ -128,6 +128,8 @@
                         <div class="misc-controls">
                             <a href="{{ route('shop.home.index') }}" class="link">{{ __('shop::app.checkout.cart.continue-shopping') }}</a>
 
+                            <a href="{{ route('cart.remove.all.items') }}" class="link">{{ __('shop::app.checkout.cart.remove-all-items') }}</a>
```

Comment:
you can use v-alert here like below

<a href="{{ route('cart.remove.all.items') }}" class="link" v-alert>{{ __('shop::app.checkout.cart.remove-all-items') }}</a>

Reply:
This will work only in bliss theme

Reply:
I'll be finished this week 🙈

Reply:
> I'll be finished this week see_no_evil

HI @ILDaviz 
Are you **working** for **Remove all cart items at once**?

Reply:
yep

Reply:
Added alert